instrumentation/grape/README.md

```diff
@@ -22,6 +31,8 @@ OpenTelemetry::SDK.configure do |c|
 end
 ```
 
+Since Grape is "designed to run on Rack or complement existing web application frameworks such as Rails and Sinatra", it is recommended to use this instrumentation it along with the Rack, Rails and/or Sinatra instrumentations.
```

Comment:
```suggestion
Since Grape is "designed to run on Rack or complement existing web application frameworks such as Rails and Sinatra", it is recommended to use this instrumentation along with the Rack, Rails and/or Sinatra instrumentations.
```

Was probably my bad, sorry!